src/components/widgets-settings/forecast-payroll/forecast-payroll.tmpl.html

```diff
@@ -4,7 +4,13 @@ <h5 translate>impac.widget.settings.forecast_payroll.title</h5>
   <div class="widget-lines-container">
     <div class="widget-line">
       {{'impac.widget.account_cash_projection.forecast_payroll.aggregate' | translate}}
-      <i ng-class="isToggleForecastPayroll() ? 'fa fa-toggle-on' : 'fa fa-toggle-off'" ng-click="toggleForecastPayroll()" />
+      <i ng-class="isToggleForecastPayroll() ? 'fa fa-toggle-on' : 'fa fa-toggle-off'"
+        ng-click="toggleForecastPayroll()" />
+    </div>
+    <div class="widget-line" ng-disabled="!isToggleForecastPayroll()" >
+      <i ng-class="isToggleTimeSheetModifier() ? 'fa fa-toggle-on' : 'fa fa-toggle-off'"
+        ng-click="toggleTimeSheetModifier()" />
+      {{'impac.widget.account_cash_projection.forecast_payroll.time_sheet_modifier' | translate}}
```

Comment:
prefer `<i translate>impac.widget.account_cash_projection.forecast_payroll.time_sheet_modifier</i>` when not interpolating with other stuff like `<i>hard coded {{'a.locale' | translate}}</i>`.